### ISSUE TITLE
Filtering requests type in the filtering log is invalid.

### ISSUE THREAD

**Issue by lancelot-moon** [AdguardTeam/AdguardForAndroid#2607]
I try v3.0.264 nightly.
For example, there are many internet requests in the filtering log.

If I only want to only view requests blocked, I configure "filter requests", but it shows nothing in the filtering log.

**Comment by lancelot-moon** [comment]
Well, must select first-party or third-party, otherwise show nothing...

**Comment by lancelot-moon** [comment]
Yes, First-party and Third-party shouldn't be mixed with Blocked, Whitelisted, Modified... etc.
https://github.com/AdguardTeam/AdguardBrowserExtension/issues/1358
<details><summary>Screenshot:</summary>

![image](https://uploads.adguard.com/slbdzd8llhm6e.png)
</details><br/>

**Comment by lancelot-moon** [comment]
@Revertron
@sfionov
@ameshkov

May you add "All" in the Request origin?
The same layout of the filter request with what AG for PC has.
Let users must choose one from All, First-party and Third-party.
Otherwise, users may untick First-party and Third-party.
<details><summary>Screenshot:</summary>

![image](https://i.imgur.com/wI3h6jQ.png)
</details><br/>

**Comment by artemiv4nov** [comment]
The filtering log looks different since v4.0 version, I'll close the task.
